packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -204,18 +204,21 @@ abstract class FlutterCommand extends Command<Null> {
 
     return new BuildInfo(getBuildMode(),
       argParser.options.containsKey('flavor')
-        ? argResults['flavor']
-        : null,
+          ? argResults['flavor']
+          : null,
```

Comment:
These should be indented by 2 (the old ones look wrong to me).

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-line-length-of-80-characters